### DISCUSSION TITLE
Whisper gives incorrect transcription - can you try?

### DISCUSSION THREAD

**Discussion by Yohan85** [openai/whisper#2279] [General]
Hi there,

On a specific track, singer starts singing at 00:26 and for some reasons Whisper gives me transcripted lyrics started from 00:00: 

` {
      "word": "Encore",
      "start": 0.0,
      "end": 11.720000267028809
    },
    {
      "word": "un",
      "start": 11.720000267028809,
      "end": 26.920000076293945
    },
    {
      "word": "jour",
      "start": 26.920000076293945,
      "end": 27.100000381469727
    },`

If you listen to the source, lyrics start at 00:26. here is the link : https://drive.google.com/file/d/1cQXgPVY05-ucFXkm3Lw0F_1kf_oUG0xg/view?usp=drive_link

Please can you try and tell me if that behaves the same way at it does for me with this mp3? Do you encoutner the same errors with your own audio?

Any help appreciated

**Comment by toanhuynhnguyen** [comment]
You can try this https://github.com/m-bain/whisperX
